dataflow/java/src/main/java/com/httparchive/dataflow/BigQueryImport.java

```diff
@@ -288,20 +318,19 @@ public static void main(String[] args) {
                 DataflowPipelineWorkerPoolOptions.AutoscalingAlgorithmType.NONE);
 
         Pipeline p = Pipeline.create(pipelineOptions);
-
-        PCollectionTuple results = p.apply(TextIO.Read
-                .named("read-har")
-                .from(getHarBucket(options))
-                .withCompressionType(TextIO.CompressionType.GZIP)
-                .withCoder(HarJsonCoder.of()))
+        PCollectionTuple results = p
```

Comment:
Added a GcsPathCoder onto the `Create.<GcsPath>...`. Now that I got past that error, it's evident that I'm not actually unzipping the har file, so I'll go back and fix that.